azurelinuxagent/ga/update.py

```diff
@@ -259,14 +259,15 @@ def run(self, debug=False):
             protocol.update_goal_state()
 
             initialize_event_logger_vminfo_common_parameters(protocol)
+            self._send_heartbeat_telemetry(protocol)
```

Comment:
Just realized we send out HeartBeat as soon as we process our first GS (after a service restart). 
Knowing that, I feel this call is redundant but I'm ok with it either way.

Reply:
if it is redundant let's remove it

Reply:
Removed